src/test/java/org/opentripplanner/raptor/rangeraptor/transit/ForwardRaptorTransitCalculatorTest.java

```diff
@@ -44,6 +47,12 @@ public void exceedsTimeLimit() {
     assertFalse(subject.exceedsTimeLimit(2_000_000_000));
   }
 
+  @Test
+  public void acceptDestinationArrival() {
+    // TODO PT: Wrire a test that test that the exceedsTimeLimit and acceptC2 predicate is
```

Comment:
```suggestion
    // TODO PT: Write a test that test that the exceedsTimeLimit and acceptC2 predicate is
```
(In case we're going to merge the comment)